### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by a89529294**
### Description

Fix typo and unnecessary code

### Additional details

fix typo in /en-US/docs/Learn/Server-side/Express_Nodejs/mongoose
remove unnecessary code in author model

### CODE REVIEW

File: files/en-us/learn/server-side/express_nodejs/mongoose/index.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ It rejects when any of the input's promises rejects, with this first rejection r
 
 The code below shows how this works.
 First we have two functions that return promises.
```
Comment (CBID2) [inline files/en-us/learn/server-side/express_nodejs/mongoose/index.md:151]:
```suggestion
First, we have two functions that return promises.
```

File: files/en-us/learn/server-side/express_nodejs/mongoose/index.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ It rejects when any of the input's promises rejects, with this first rejection r
...
-We `await` on both of them to compete using the promise returned by `Promise.all()`.
+We `await` on both of them to complete using the promise returned by `Promise.all()`.
 Once they both complete `await` returns and the results array is populated.
 The function then continues to the next `await`, and waits until the promise returned by `anotherFunctionThatReturnsPromise()` is settled.
```
Comment (CBID2) [inline files/en-us/learn/server-side/express_nodejs/mongoose/index.md:154]:
```suggestion
Once they both complete `await` returns and the results array is populated,
the function then continues to the next `await`, and waits until the promise returned by `anotherFunctionThatReturnsPromise()` is settled.
```

**Review by CBID2 — COMMENTED** [review]
Hi @a89529294! :) I found some more typos in your PR and provided suggestions on how to fix them. Other than that, you're off to a great start! :)

File: files/en-us/learn/server-side/express_nodejs/mongoose/index.md (hunk 2 above)
Comment (a89529294) [reply files/en-us/learn/server-side/express_nodejs/mongoose/index.md]: Add both suggestions

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

This PR looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! ✌️ 

If you're looking for something else to help out with, we have a [contributor board](https://github.com/orgs/mdn/projects/25), and another useful place to look is the list of issues labeled [`good first issue`](https://github.com/mdn/content/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

**Pull request merged**